slick-testkit/src/codegen/resources/dbs/mysql.sql

```diff
@@ -1 +1,3 @@
-CREATE TABLE LongTextTest( entry1 LONGTEXT NOT NULL , entry2 MEDIUMTEXT NOT NULL , entry3 TEXT NOT NULL , entry4 VARCHAR(255) NOT NULL );
+CREATE TABLE `LongTextTest`( entry1 LONGTEXT NOT NULL , entry2 MEDIUMTEXT NOT NULL , entry3 TEXT NOT NULL , entry4 VARCHAR(255) NOT NULL );
+
+CREATE TABLE `table_name`( id INT NOT NULL );
```

Comment:
There's some Unicode screw-up in this line that makes the build fail on my machine. Some of the spaces on this line are actually non-breaking spaces. (maybe some MySQL versions are OK with that or there's a weird conversion happening when checking out in CI)